docs/tutorials/basic/python.md

```diff
@@ -376,21 +372,21 @@ Calling the response-streaming `ListFeatures` is similar to working with
 sequence types:
 
 ```python
-for feature in stub.ListFeatures(rectangle, timeout_in_seconds):
+for feature in stub.ListFeatures(rectangle):
 ```
 
 #### Request-streaming RPC
 
-Calling the request-streaming `RecordRoute` is similar to passing a sequence to
-a local method. Like the simple RPC above that also returns a single response,
-it can be called synchronously or asynchronously:
+Calling the request-streaming `RecordRoute` is similar to passing an iterator
+to a local method. Like the simple RPC above that also returns a single
+response, it can be called synchronously or asynchronously:
 
 ```python
-route_summary = stub.RecordRoute(point_sequence, timeout_in_seconds)
+route_summary = stub.RecordRoute(iter(point_sequence))
```

Comment:
What do you think of renaming the variable to point_iterator to express this instead?